src/_P064_APDS9960.ino

```diff
@@ -65,14 +71,15 @@ boolean Plugin_064(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_GET_DEVICEVALUENAMES:
       {
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_064));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_064));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[2], PSTR(PLUGIN_VALUENAME3_064));
-        /*
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[3], PSTR(PLUGIN_VALUENAME4_064));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[4], PSTR(PLUGIN_VALUENAME5_064));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[5], PSTR(PLUGIN_VALUENAME6_064));
-        */
+        if (PCONFIG(1) == PLUGIN_MODE_GPL_064) { // Gesture/Proximity/ALS mode

```

Comment:
This check is done in a number of places.
Maybe add a define for it, like `#define P064_IS_GPL_SENSOR (PCONFIG(1) == PLUGIN_MODE_GPL_064)`

Reply:
Done. It sure improves readability 👍